hack/docker_build.sh

```diff
@@ -40,7 +40,7 @@ if [[ $? != 0 ]]; then
   echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
 fi
 
-TAG=$TAG-multiarch
+TAG=$TAG
```

Comment:
I think this line should be removed.

`TAG=$TAG` has no meaning.

Reply:
right, removed this line . it was unexpected to add to this file